restapi/src/main/java/io/stargate/web/docsapi/resources/DocumentResourceV2.java

```diff
@@ -633,6 +639,19 @@ public Response getDocPath(
             }
           }
 
+          // check first that namespace and table exist
+          AuthenticatedDB authenticatedDB = dbFactory.getDataStoreForToken(authToken, allHeaders);
+          // TODO does this require authenticateSchemaRead?
+          Keyspace keyspace = authenticatedDB.getKeyspace(namespace);
```

Comment:
I think not because here we just validating the name. I assume an `authorizeDataRead` check is executed further down the call path. `authorizeSchemaRead` is required when we are exposing information about the schema, like listing objects of "describing", which here it is just a data read. @EricBorczuk , @dougwettlaufer please correct me if I'm wrong.

Reply:
OK, removing TODO for now.